README.md

```diff
@@ -48,7 +48,7 @@ Import the `faculty-distributed` module and find the faculty platform project ID
 import faculty_distributed
 import os
 
-project_id = os.getenv['project_id']
+project_id = os.getenv("FACULTY_PROJECT_ID")
```

Comment:
If the downstream code doesn't accept `project_id` to be `None`, I'd suggest raising an exception here if the environment variable isn't found:
```suggestion
project_id = os.environ["FACULTY_PROJECT_ID"]
```